examples/kind/README.md

```diff
@@ -0,0 +1,179 @@
+    The push refers to repository [localhost:5000/app]
+    8850e2de6704: Pushed
+    f4da1b8b450d: Layer already exists
+    c2b2d07a43bf: Layer already exists
+    1c60dde6f5e0: Layer already exists
+    0a454283a2a4: Layer already exists
+    0251486b80af: Layer already exists
+    8940ae814902: Layer already exists
+    678a0785e7d2: Layer already exists
+    NsZqtco: digest: sha256:96550aba92e3df23bc0f8b6f70fbba3e573a11d3311cd05342f02b92f211b6fa size: 1992
+    [info] Image pushed to registry (localhost:5000)
+    [done] √ Done processing image 'localhost:5000/app'
+    [info] Execute 'helm list --namespace test-1 --output json --kube-context kind-kind'
+    [info] Execute 'helm upgrade quickstart /Users/pratikjagrut/.devspace/component-chart/component-chart-0.8.1.tgz --namespace test-1 --values /var/folders/9c/mv8m0y2j5_13zyd93yxwl71m0000gn/T/667828839 --install --kube-context kind-kind'
+    [info] Execute 'helm list --namespace test-1 --output json --kube-context kind-kind'
+    [done] √ Deployed helm chart (Release revision: 1)
+    [done] √ Successfully deployed quickstart with helm
+
+    #########################################################
+    [info] DevSpace UI available at: http://localhost:8090
+    #########################################################
+
+    [done] √ Port forwarding started on 3000:3000 (test-1/quickstart-55687985f-h9nqm)
+    [0:sync] Waiting for pods...
+    [0:sync] Starting sync...
+    [0:sync] Inject devspacehelper into pod test-1/quickstart-55687985f-h9nqm
+    [0:sync] Start syncing
+    [0:sync] Sync started on /Users/pratikjagrut/work/github.com/loft-sh/devspace/examples/kind <-> . (Pod: test-1/quickstart-55687985f-h9nqm)
+    [0:sync] Waiting for initial sync to complete
+    [0:sync] Helper - Use inotify as watching method in container
+    [0:sync] Downstream - Initial sync completed
+    [0:sync] Upstream - Upload File 'Dockerfile'
+    [0:sync] Upstream - Upload File 'devspace.yaml'
+    [0:sync] Upstream - Upload 2 create change(s) (Uncompressed ~0.57 KB)
+    [0:sync] Upstream - Successfully processed 2 change(s)
+    [0:sync] Upstream - Initial sync completed
+    [info] Starting log streaming
+    [quickstart] Start streaming logs for test-1/quickstart-55687985f-h9nqm/container-0
+    [quickstart]
+    [quickstart] > node-js-sample@0.0.1 start /app
+    [quickstart] > nodemon index.js
+    [quickstart]
+    [quickstart] [nodemon] 2.0.12
+    [quickstart] [nodemon] to restart at any time, enter `rs`
+    [quickstart] [nodemon] watching path(s): *.*
+    [quickstart] [nodemon] watching extensions: js,mjs,json
+    [quickstart] [nodemon] starting `node index.js`
+    [quickstart] Example app listening on port 3000!
+    ```
+
+**Goto browser and hit http://localhost:3000**
+
+You'll see
+
+```sh
+Hello World!
+```
+
+- **Make changes to source code and see the devspace magic**
+
+    Change line `res.send('Hello World!');` to `res.send('Hello K8S, it\'s DevSpace!');` and save the file.
```

Comment:
The line is already `Hello K8S, it\'s DevSpace!`, it would be probably also better to use `res.send("Hello K8S, it's DevSpace!");`